docs/source/conf.py

```diff
@@ -54,7 +54,7 @@
 
 # General information about the project.
 project = u'Planet API Client'
-copyright = u'2017-2019 PLANET LABS INC. ALL RIGHTS RESERVED. MADE IN SAN FRANCISCO. 645 HARRISON STREET, SAN FRANCISCO 94107'
+copyright = u'2017-2022 PLANET LABS INC. ALL RIGHTS RESERVED. MADE IN SAN FRANCISCO. 645 HARRISON STREET, SAN FRANCISCO 94107'
```

Comment:
We need to change this to be PBC instead of INC.

```suggestion
copyright = u'2017-2021 PLANET LABS INC., 2021-2022 PLANET LABS PBC. ALL RIGHTS RESERVED. MADE IN SAN FRANCISCO. 645 HARRISON STREET, SAN FRANCISCO 94107'
```